zppy/templates/pcmdi_diags/cmip_variables.json

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
The `pcmdi_diags` subdirectory should be moved under `inclusions/` and paths should be updated accordingly.